daml-lf/transaction-test-lib/src/main/scala/lf/transaction/test/TransactionBuilder.scala

```diff
@@ -185,7 +185,7 @@ object TransactionBuilder {
       consuming: Boolean,
       actingParties: Set[String],
       argument: Value,
-      byKey: Boolean = true,
+      byKey: Option[Boolean] = Some(false),
```

Comment:
Flipping the default from `true` to `Some(false)` seems strange to me. Having a default value for this at all seems like a giant foot gun anyway. I think we should force the callers to think about how they are exercising.

Reply:
Thank you.